pkg/console/operator/sync_v400.go

```diff
@@ -221,8 +226,8 @@ func (co *consoleOperator) SyncDeployment(
 	proxyConfig *configv1.Proxy,
 	canMountCustomLogo bool) (consoleDeployment *appsv1.Deployment, changed bool, reason string, err error) {
 
+	updatedOperatorConfig := operatorConfig.DeepCopy()
 	requiredDeployment := deploymentsub.DefaultDeployment(operatorConfig, cm, serviceCAConfigMap, defaultIngressCertConfigMap, trustedCAConfigMap, sec, proxyConfig, canMountCustomLogo)
-	expectedGeneration := getDeploymentGeneration(co)
```

Comment:
`getDeploymentGeneration` is now unused, but I'd prefer to get this PR merged in current form and leave that cleanup to future efforts.

Reply:
Yup, we have sevearl cleanup items we will defer to get you unblocked.